### DISCUSSION TITLE
建议全局规则只作用于用户app

### DISCUSSION THREAD

**Discussion by jiuqianyuan** [gkd-kit/gkd#464] [Ideas]
1、建议读取应用列表，全局规则只对用户安装的app生效(非系统应用、系统框架应用)；
2、全局规则内 改为类似"应用规则"分类 的可视化应用列表 开关。
3、"默认应用/应用规则"或以后全局规则的 app列表，未安装应用默认不显示。

![IMG_20240107_210940](https://github.com/gkd-kit/gkd/assets/39406781/1a7d60a2-fec7-4766-96e2-ddd207f13acc)

**Comment by lisonge** [comment]
我看看

**Comment by lisonge** [comment]
https://github.com/gkd-kit/gkd/commit/7768bd47327a9900e855a8c7a16d6c8683a15c86
